src/test/java/com/google/api/codegen/discogapic/testdata/java/java_simplecompute.v1.json.baseline

```diff
@@ -8587,6 +8611,9 @@ import javax.annotation.Nullable;
 
 @Generated("by GAPIC")
 @BetaApi
+/**
+ * Request object for method compute.addresses.patch. Updates an address in the specified project using the data included in the request. This method supports PATCH semantics and uses the JSON merge patch format and processing rules.
```

Comment:
I understand the line formatting is invoked by artman post-generator, correct?

Reply:
correcto